plugins/webhooks/index.js

```diff
@@ -62,9 +62,11 @@ const webhooksPlugin = {
                     maxBytes: parseInt(pluginOptions.maxBytes, 10) || DEFAULT_MAX_BYTES
                 },
                 handler: async (request, h) => {
-                    const { pipelineFactory } = request.server.app;
+                    const { pipelineFactory, queueWebhook } = request.server.app;
                     const { scm } = pipelineFactory;
+                    const { executor, queueWebhookEnabled } = queueWebhook;
                     let message = 'Unable to process this kind of event';
+                    let hookId;
```

Comment:
Although it is not related to this feature addition, the hookId could not be referenced in the `catch` scope, so I fixed it as well.